main.tf

```diff
@@ -66,6 +67,27 @@ resource "aws_eip" "sftp" {
   vpc = local.is_vpc
 }
 
+module "security_group" {
+  count = local.create_sg ? 1 : 0
+
+  source  = "cloudposse/security-group/aws"
+  version = "0.3.1"
+
+  vpc_id = local.is_vpc ? var.vpc_id : null
+
+  rules = [
+    {
+      type        = "ingress"
+      from_port   = 22
+      to_port     = 22
+      protocol    = "tcp"
+      cidr_blocks = var.allowed_cidrs
+    }
```

Comment:
Nono, this goes against the spirit of the module. The objective is to expsoe the `rules` variable so the caller has unfettered ability to add rules for ingress/egress.